### PR TITLE
New tsv parser for interproscan files 

### PR DESCRIPTION
**Pull request by Zygnematophyce**
Hi @holmrenser,
In this new PR, I focused on new functionality for interproscan, I added:
- The ability to read an interproscan output file in **.tsv format** and update the database,
- The interproscan CLI command **add interproscan** becomes a job queue,
- Moreover, the files are read line by line split by tabulation, to avoid errors in loading files that are too large (> 512 MB with fs.readFileSync) in memory.

### CODE REVIEW

File: cli/genenotebook.js [hunk 1]
```diff
@@ -356,35 +356,68 @@ addTranscriptome
...
+
+    let parserType;
+    const extensionFile = path.extname(file).replace(/\./g, '');
+
+    if (['tsv', 'gff3'].includes(parser)) {
+      parserType = parser;
+    } else if (['tsv', 'gff3'].includes(extensionFile)) {
+      parserType = extensionFile;
+    } else {
+      logger.error('--format parameter is not defined');
```
Comment (holmrenser) [inline cli/genenotebook.js:404]: It seems like this is only an error when both `parser` is undefined and the file suffix is not `gff3` or `tsv`. The error message should probably reflect that, something like 'Must specify --format when file extension is not tsv or gff3'.

**Comment by holmrenser** [comment]
Awesome, I'm reviewing now and hope to finish today/tomorrow. So far looks good!

File: cli/genenotebook.js (hunk 1 above)
Comment (Zygnematophyce) [reply cli/genenotebook.js]: No problem, I just fixed that. And add more precise indications when the error occurs.

**Review by holmrenser — APPROVED** [review]
lgtm!

**Pull request merged**